site-packages/integralstor_utils/filesize.py

```diff
@@ -57,13 +57,35 @@ def to_bytes(sizes_list):
     args:        each entry in sizes_list must be of format 'value unit' (100 M)
     returns:     a list with entries in bytes, no suffix attached.
     """
-
+    import re
     in_bytes = []
     try:
-        defs = {'KB': 1024, 'MB': 1024**2, 'GB': 1024**3, 'TB': 1024 **
-                4, 'K': 1024, 'M': 1024**2, 'G': 1024**3, 'T': 1024**4}
+        regexes = [
+            "^[0-9]*\.?[0-9]* (KB|K)$",
+            "^[0-9]*\.?[0-9]* (MB|M)$",
+            "^[0-9]*\.?[0-9]* (GB|G)$",
```

Comment:
Would have been simpler if the regex was made just one string like `"^[0-9]*\.?[0-9]* (KB|K|MB|M|GB|G|TB|T)$"`. Should I put it as one?